src/Views/OperatingSystemView.vala

```diff
@@ -373,6 +373,12 @@ public class About.OperatingSystemView : Gtk.Box {
                         details.packages.length
                     ).printf (details.packages.length);
 
+                    foreach (unowned var info in details.info) {
```

Comment:
I think you can simplify this to `if (Pk.Info.SECURITY in details.info)`

Reply:
You're so smart!